packages/types/src/fulfillment/common/shipping-profile.ts

```diff
@@ -0,0 +1,11 @@
+import { ShippingOptionDTO } from "./shipping-option"
+
+export interface ShippingProfileDTO {
```

Comment:
thought(non-blocking): Perhpas, it would be convenient to have an open-ended type field on the ShippingProfile model to group options easily. E.g. in the core our default types are `default` and `gift_card`.

Reply:
But this DTO don't be used by the core right, this is suppose to represent the module shipping profile. it wouldn't be usable for the current core if this DTO would have properties that does not exists in the core and would provide wrong information. wdyt?

Reply:
I am not sure I understand. I was just proposing adding a `type` field to the shipping profile model in the new fulfillment module, so shipping options are grouped under a type on the profile. Instead of relying on the name to indicate the type.

Reply:
Humm got it, i though it would be covered by the fulfillment set type column since a profile is linked to an option which is part of a fulfillment set

Reply:
got it, we can still add it later in the process 👍